lib/ferrum/node.rb

```diff
@@ -46,6 +46,8 @@ def click(mode: :left, keys: [], offset: {}, delay: 0)
       modifiers = page.keyboard.modifiers(keys)
 
       case mode
+      when :force
+        evaluate("this.click()")
```

Comment:
it's based on real clicks: 

> Puppeteer also provides two methods to deliver real clicks to the elements:page.click('.foo') and elementHandle.click(). These, however, have a little bit of trickery underneath.

https://medium.com/@aslushnikov/automating-clicks-in-chromium-a50e7f01d3fb

and on the `force: true` option in Cypress, for example:
```
cy.get('input[type="search"]').click({force: true})
```

Reply:
This one is synthetic event and has nothing in common with real clicks